protocol-dashboard/src/App.tsx

```diff
@@ -27,8 +27,9 @@ import NotFound from 'containers/NotFound'
 import Proposal from 'containers/Proposal'
 import { createStyles } from 'utils/mobile'
 import { getDidGraphError } from 'store/api/hooks'
+import UnregisteredNode from 'containers/UnregisteredNode'
 const styles = createStyles({ desktopStyles, mobileStyles })
-const history = createBrowserHistory({
+const history = createHashHistory({
```

Comment:
is this the fix for the problematic refresh (from https://dashboard.staging.audius.org/services/content-node to https://dashboard.staging.audius.org/#/./services/content-node) on stage and prod? a little confused here because this doesn't happen on DNs at /dashboard, and doesn't seem like an IPFS-specific issue because only prod is on IPFS. wondering if its possible to keep browser history over hash history here

Reply:
since it's a webserver on DN that would be more compatible with browser history than IPFS, but i was seeing the refresh issue locally as well. i think this is what fixed the refresh when running locally, but i can test again with browser history and see if it's an issue

Reply:
[this](https://github.com/AudiusProject/audius-protocol/blob/main/protocol-dashboard/scripts/rerouteLegacy.cjs#L6) is what's messing it up! i'll look more to see what we can do. it looks like stage is running via `npm run start:stage`, so this step doesn't affect it. i can repro the refresh bug when running `npm run build:stage && npm run serve` locally

Reply:
nice, thanks for looking into it more! ill approve for now since you got it all working and im happy to take this follow up task to ideally remove the hash routing as that would be best for the DN situation... which you shouldnt have to worry about